src/index.ts

```diff
@@ -170,7 +174,7 @@ async function resolveTarStreamOnFinish(
 			new Error(
 				dockerfile
 					? `Specified dockerfile could not be resolved: ${dockerfile}`
-					: 'Resolution could not be performed',
+					: noBuildFilesFoundErrorString,
```

Comment:
While at it, I suggest amending the `Specified dockerfile could not be resolved` message as well, mainly to avoid "could not be resolved" as it may not be obvious that it means "file not found". Suggestion: `File not found: "${dockerfile}"`